snowplow_tracker/tracker.py

```diff
@@ -720,16 +736,26 @@ def track_screen_view(
         :rtype:                 tracker
         """
         screen_view_properties = {}
+
+        screen_view_properties["id"] = id_
+
         if name is not None:
             screen_view_properties["name"] = name
-        if id_ is not None:
-            screen_view_properties["id"] = id_
+        if type is not None:
+            screen_view_properties["type"] = type
+        if previous_name is not None:
+            screen_view_properties["previousName"] = previous_name
+        if previous_id is not None:
+            screen_view_properties["previousId"] = previous_id
+        if previous_type is not None:
+            screen_view_properties["previousType"] = previous_type
+        if transition_type is not None:
+            screen_view_properties["transitionType"] = transition_type
 
         event_json = SelfDescribingJson(
-            "%s/screen_view/%s/1-0-0" % (BASE_SCHEMA_PATH, SCHEMA_TAG),
+            "%s/screen_view/%s/1-0-0" % (MOBILE_SCHEMA_PATH, SCHEMA_TAG),
             screen_view_properties,
         )
```

Comment:
This is a nice change, but its a big breaking change for anyone using it in <= 0.12.0. 

I wonder if we should leave a deprecated version of the old tracking method in until v1? As we're in v0.x we can introduce a `_legacy` version perhaps? (There might be a better name here!)